react/week2/todo/src/ListOfItem.js

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from "react";
+import DeleteIcon from "@material-ui/icons/Delete";
+const ListOfItem = (prop) => {
+  const [line, setLine] = useState(false);
+  const deleteItem = () => {
```

Comment:
you can delete the item by give it an ID and then use `.filter()` function in JS, for example:

```
const deleteItem = (id) => {
    setItems(items.filter((item) => item.id !== id));
  };
```
and better experience, we could add to the item a bolean attributes (`done`), and reverse its value on every check. 
 example: 
```
const handleCheck = (id) => {
    const newItems = Items.map((item) => {
      if (item.id === id) {
        return { ...item, done: !item.done };
      }
      return item;
    });
    setItems(newItems);
  };
```